static/js/gear_menu.js

```diff
@@ -144,6 +146,18 @@ export function close() {
 
 export function set_avatar() {
     const avatar_url = page_params.avatar_url;
-
-    $("#gear-menu-avatar").attr("src", avatar_url);
+    const user_id = page_params.user_id;
+    const full_name = page_params.full_name;
+    const active_status = presence.get_status(user_id);
+    const last_seen = buddy_data.user_last_seen_time_status(user_id);
+    const user_circle_class = buddy_data.get_user_circle_class(user_id);
+
+    $(".settings-dropdown-avatar").attr("src", avatar_url);
```

Comment:
Why not directly set the avatar using this class in the first commit?

Reply:
I might have tested this with the id in the first commit and added and pushed that as well. Will change that in the first commit itself.